### PR TITLE
Add Dwapi metrics endpoint from psmart module

### PR DESCRIPTION
**Pull request by ojwanganto**
DWAPI metrics endpoint will be available at `<host>/openmrs/ws/rest/v1/kenyaemr/getemrmetrics`
The response looks like the below:
`{
    "EmrName": "KenyaEMR",
    "EmrVersion": "19.1.2",
    "LastLoginDate": "2024-Jul-26",
    "LastMoH731RunDate": null
}
`

@andrineM @Murithijoshua @RAJABIBRAZ please note that we'll not need to include the psmart module in our releases.

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/kenyaemr/web/controller/KenyaemrCoreRestController.java [hunk 1]
```diff
@@ -2854,6 +2855,23 @@ public List<org.openmrs.module.webservices.rest.SimpleObject> search(@RequestPar
...
+    @RequestMapping(method = RequestMethod.GET, value = "/getemrmetrics")
+    @ResponseBody
+    public Object getKenyaEMRDetails(HttpServletRequest request) {
+
+        return "{\n" +
+                "    \"EmrName\":\"KenyaEMR\",\n" +
+                "    \"EmrVersion\":\"" + DwapiMetricsUtil.getKenyaemrVersion() + "\",\n" +
+                "    \"LastLoginDate\":\"" + DwapiMetricsUtil.getLastLogin() + "\",\n" +
+                "    \"LastMoH731RunDate\":\"" + DwapiMetricsUtil.getDateofLastMOH731() + "\"" +
+                "}";
```
Comment (makombe) [inline omod/src/main/java/org/openmrs/module/kenyaemr/web/controller/KenyaemrCoreRestController.java:2872]: What do you think if we use the` SimpleObject so = new SimpleObject()` here? could be cleaner.

**Review by makombe — APPROVED** [review]

**Comment by Murithijoshua** [comment]
@ojwanganto  this is noted

File: omod/src/main/java/org/openmrs/module/kenyaemr/web/controller/KenyaemrCoreRestController.java (hunk 1 above)
Comment (ojwanganto) [reply omod/src/main/java/org/openmrs/module/kenyaemr/web/controller/KenyaemrCoreRestController.java:2872]: Makes sense. I just lifted the previous implementation. I will address this.

**Comment by ojwanganto** [comment]
@makombe - I have addressed the reviews. Thanks

**Comment by makombe** [comment]
> @makombe - I have addressed the reviews. Thanks

Thanks

**Pull request merged**